front/src/common/RollingStockSelector/RollingStockSelector.tsx

```diff
@@ -1,37 +1,38 @@
-import React, { useEffect, useRef, useState } from 'react';
-import PropTypes from 'prop-types';
-import { useSelector } from 'react-redux';
-import { get } from 'common/requests';
-import { useTranslation } from 'react-i18next';
+import React, { useRef } from 'react';
 import RollingStockModal from 'common/RollingStockSelector/RollingStockModal';
 import icon from 'assets/pictures/components/train.svg';
 import RollingStock2Img from 'common/RollingStockSelector/RollingStock2Img';
 import {
   comfort2pictogram,
   RollingStockInfos,
 } from 'common/RollingStockSelector/RollingStockHelpers';
-import { getRollingStockID, getRollingStockComfort } from 'reducers/osrdconf/selectors';
 import { useModal } from 'common/BootstrapSNCF/ModalSNCF';
+import { RollingStock } from 'common/api/osrdEditoastApi';
+import exampleRollingStockImage1 from 'assets/defaultRSImages/example_rolling_stock_image_1.gif';
+import exampleRollingStockImage2 from 'assets/defaultRSImages/example_rolling_stock_image_2.gif';
 
-const ROLLINGSTOCK_URL = '/rolling_stock';
+type RollingStockProps = {
+  rollingStockID?: number;
+  rollingStockComfort?: string;
+  rollingStockSelected?: RollingStock;
+  comfort?: string;
+  comfortType?: string;
+  choice?: string;
```

Comment:
```suggestion
  comfortTranslation?: string;
  comfortTypeTranslation?: string;
  choiceTranslation?: string;
```
I think we should take some time to discuss about the translation in storybook